data/avro-schema/src/main/resources/avro/net/corda/data/p2p/GatewayTlsCertificates.avsc

```diff
@@ -8,6 +8,11 @@
       "name": "tenantId",
       "type": "string"
     },
+    {
+      "doc": "The holding identity that the certificate belongs to",
```

Comment:
maybe better to say here "that the certificate is used by"?

Since the certificate can be a cluster-level certificate, shared by many vnodes (i.e. not really belonging to any of them specifically).

Reply:
Should we also have a field indicating if it's a client or server certificate?